src/main/java/com/minecolonies/util/LanguageHandler.java

```diff
@@ -62,8 +63,10 @@ public static String getString(String key)
      * @param defaultValue the value to return if no key is found
      * @return Localized string
      */
+    @SuppressWarnings("deprecation")
```

Comment:
We are suppose to be doing translations on the client now. net.minecraft.client.resources.I18n Which we can do, I'll look at it later